src/summersplash2022/index.module.css

```diff
@@ -1142,204 +1142,96 @@
   }
 }
 
-@media (min-width: 2000px) {
-  /* 900px*/
-  .scrollContainer,
-  .section1,
-  .section2,
-  .section3,
-  .section5,
-  .whyWorkAtVariant,
-  .section6 {
-    height: auto;
-    scroll-behavior: smooth;
-  }
-
-  .scrollContainer {
-    scroll-behavior: smooth;
-    scroll-snap-type: y mandatory;
-    overflow-y: scroll;
-  }
-
-  /* Headings */
-  .section3Heading,
-  .section5Heading,
-  .section6Heading,
-  .section4Heading,
-  .mostImportantForYou {
-    font-family: 'Recoleta';
-    font-style: normal;
-    font-weight: 400;
-    font-size: 200%;
-    line-height: 39px;
-  }
-
-  /* Section 1 */
-  .imageBlob1 {
-    position: relative;
-    left: 30%;
-    height: auto;
-    bottom: 220px;
-    width: 50%;
+@media (min-width: 2150px) {
+  .heading,
+  .handbookHeaderDeskotop {
+    font-size: 300%;
+    line-height: 200%;
```

Comment:
Dette er mer som en ting til senere, men ville i utgangspunktet prøvd å bruke rem i stedet for % på fontstørrelser. Tror det finnes noen usecases for em/%, men er veldig lett at det blir litt forvirrende behavior på tekststørrelser da em/% regnes ut med at man tar utgangspunkt i fontstørrelsen til elementet de brukes på, mens rem tar utgangspunkt i størrelsen i root-elementet. Det gjør at det blir en god del mer consistent.

Tror nok det finnes en del use cases for em/% også, men har ikke opplevd at jeg har hatt noe behov for å bruke det selv.

(1 rem/em = 100% i fontstørrelse, btw. Although forskjellene på rem kontra em/% ofc også spiller en rolle).